src/amo/components/AddonMeta.js

```diff
@@ -8,24 +8,54 @@ import 'amo/css/AddonMeta.scss';
 
 export class AddonMetaBase extends React.Component {
   static propTypes = {
-    averageDailyUsers: PropTypes.number.isRequired,
+    addon: PropTypes.object.isRequired,
     i18n: PropTypes.object.isRequired,
   }
 
   render() {
-    const { averageDailyUsers, i18n } = this.props;
+    const { addon, i18n } = this.props;
+    const averageDailyUsers = addon.average_daily_users;
+    const averageRating = addon.ratings.average;
+    const addonReviewCount = addon.ratings.count;
 
     const userCount = i18n.sprintf(
       i18n.ngettext('%(total)s user', '%(total)s users', averageDailyUsers),
       { total: i18n.formatNumber(averageDailyUsers) },
     );
+    let starCount;
+    if (averageRating) {
+      starCount = i18n.sprintf(
+        i18n.gettext('%(averageRating)s out of 5'),
+        { averageRating: i18n.formatNumber(averageRating) },
+      );
+    } else {
+      starCount = i18n.gettext('Not yet rated');
```

Comment:
This is pedantic but I wonder if `'No ratings'` (and `'No reviews'`, below) would be a bit easier to grok and read. Not yet rated feels almost like an invitation to action ("YOU could be the first to rate it") but here it really isn't and I think it's weirder to parse as-is.

Could we change it?